src/Popover/Popover.js

```diff
@@ -64,9 +64,14 @@ class Popover extends React.PureComponent<Props, State> {
             </Overlay>
 
             {/* Note: Arrow */}
-            <Overlay show={isOpen} resize target={target} {...position.arrow}>
+            <StyledOverlay
```

Comment:
NOTE: do not wrap `StyledCard` (L61) with `StyledOverlay`;
let user handles `onHide` by theirselves